### PR TITLE
Cleanup steps after CRI introduction, information model, packed CBOR and no-text-format

### PR DESCRIPTION
**Pull request by chrysn**
Remnants of versions before these were littered all over the place; I went through the document and changed things in what I hope to be managable commits.

(Created as a draft as it's still incomplete; especially I'd like to move the information model around and deduplicate against what was the precursor to the information model).

### CODE REVIEW

**Comment by chrysn** [comment]
I didn't get around to performing *all* the clean-up yet, leaving the model duplication for now in order to create a publishable as-good-as-it-is version.

File: draft-ietf-core-coral.md [hunk 1]
```diff
@@ -149,26 +148,10 @@ such as the [Hypertext Transfer Protocol (HTTP)](#RFC7230) or the
 [Constrained Application Protocol (CoAP)](#RFC7252).
 
 This document defines the CoRAL data model and interaction model as well
-as two specialized CoRAL serialization formats.
+as a compatct serialization format.
```
Comment (cabo) [inline draft-ietf-core-coral.md:151]: typo

File: draft-ietf-core-coral.md [hunk 2]
```diff
@@ -1194,44 +1173,38 @@ Applications may also mandate the following and other restrictions:
...
+URIs that identify vocabulary do not need to be registered. The
+inclusion of domain names in URIs allows for the decentralized
+creation of new URIs without the risk of collisions.
 
-However, IRIs can be relatively verbose and impose a high overhead on
+However, URIs can be relatively verbose and impose a high overhead on
 a representation.
 This can be a problem in [constrained environments](#RFC7228).
-Therefore, CoRAL alternatively allows the use of unsigned integers to
+Therefore, CoRAL alternatively allows the use of packed entities to
```
Comment (cabo) [inline draft-ietf-core-coral.md:1207]: Is "packed entities" a term?
(Surely OK for now, but really they are references, no?)

File: draft-ietf-core-coral.md (hunk 1 above)
Comment (chrysn) [reply draft-ietf-core-coral.md:151]: Thanks, fixed.

File: draft-ietf-core-coral.md (hunk 2 above)
Comment (chrysn) [reply draft-ietf-core-coral.md:1207]: "Packed reference" is the term that should have been used. Thanks, hotfixed.

**Pull request merged**